Arena/terrain.py

```diff
@@ -0,0 +1,31 @@
+from pygame import Vector2
+import pygame
+
+
+class Terrain():
+    def __init__(self) -> None:
+        self.size = Vector2(50, 50)
```

Comment:
This should either be passed to the `Terrain` class via game state, or removed from the game state, to avoid code duplication.